mxcube3/ui/components/SampleView/MotorInput.js

```diff
@@ -65,6 +66,7 @@ export default class MotorInput extends React.Component {
                 step={step}
                 defaultValue={valueCropped}
                 name={motorName}
+                disabled={this.props.state === 4 || this.props.state === 3 || this.props.state <= 1}
```

Comment:
Er... I tried this morning, first with this code (with the `disabled`) and I could not find any problem with it and then I removed the `disabled` attribute, I could type in the input field for the motors in fault state (Kappa ones) and I could type while motor was moving.
I am sure I am missing something ?